tools/filters/gff/gtf_filter_by_attribute_values_list.xml

```diff
@@ -1,8 +1,8 @@
 <tool id="gtf_filter_by_attribute_values_list" name="Filter GTF data by attribute values_list" version="0.1">
```

Comment:
Do we want to upgrade the tool version here? I lean toward yes here?

Reply:
done